test/fika/parser_test.exs

```diff
@@ -44,13 +44,36 @@ defmodule Fika.ParserTest do
     # ! -         Unary           Boolean not and arithmetic unary minus
     # * /         Left            Arithmetic binary mult and div
     # + -         Left            Arithmetic binary plus and minus
-    # < > <= >=   Left            Comparison                                Implement
-    # == !=       Left            Comparison                                Implement
+    # < > <= >=   Left            Comparison
+    # == !=       Left            Comparison
     # &           Left            Boolean AND
     # |           Left            Boolean OR
     # =           Right           Match operator                            Parser refactoring
     # &           Unary           Capture operator                          Parser refactoring
 
+    # These operators cannot be combined to form a valid expression but we test the parser anyway
```

Comment:
Previously, I "injected" the `!`operator just before the `&`operator, testing that `&` has less precedence than `!`.
I think it was cleaner, but also more confusing. What do you think?